src/Http/WebUtilities/src/HttpResponseStreamWriter.cs

```diff
@@ -127,6 +128,30 @@ public override void Write(char[] values, int index, int count)
             }
         }
 
+        public override void Write(ReadOnlySpan<char> values)
```

Comment:
```suggestion
        public override void Write(ReadOnlySpan<char> value)
```

Unless I'm mistaken, this should just be a single value (parallel to Write(string))

Reply:
Same with other overloads.

Reply:
Done!
FYI I had used `values` as it was used for the `char[]` overload. I also didn't use `buffer` like in the base class as otherwise it could have been confused with the internal buffer.